language/move-prover/tests/sources/invariants.exp

```diff
@@ -23,43 +23,6 @@ error:  This assertion might not hold.
     =     at tests/sources/invariants.move:54:20: invalid_R_update
     =         r = <redacted>
 
-error:  This assertion might not hold.
-
```

Comment:
That's not right and I think what we forget here is that even with spec blocks, functions can still have invariants via resources or on module level.

We need to undo this flag change. I come back and look at more thorough solution in a different PR.